tests/metaworld/envs/mujoco/sawyer_xyz/scripted_policies_utils.py

```diff
@@ -0,0 +1,77 @@
+"""Implements functions for creating scripted policies for task environments.
+
+examples of functions are move_end_effector, close_gripper, open_gripper, etc.
+"""
+import numpy as np
+
+
+def move_end_effector(env, pos, quat=None, num_sim_steps=100, render=False):
+    """Move the position of the end effector to goal_location.
+
+    Args:
+        env (Metaworld.envs.mujoco.SawyerXYZEnv): The mujoco-py sawyer
+            environment.
+        pos (numpy.ndarray): The location to reposition the end
+            effector to. Shape is `(3,)`
+        quat (numpy.ndarray): The location to reposition the end
+            effector to. Shape is `(4,)`
+        num_sim_steps(int): The number of steps taken within the mujoco
+            simulator.
+        render(bool): Whether or not the environment should be render after
+            the end effector has been moved.
+
+    """
+    env.data.set_mocap_pos('mocap', pos)
```

Comment:
sorry but you can't call these methods. you should only call env.step()

Reply:
Ok I'll need to talk to @avnishn to figure out how to change it. He gave me the `scripted_policies_utils.py` file from his branch and told me to work off of it.